packages/form-core/src/FormApi.ts

```diff
@@ -562,7 +562,7 @@ export class FormApi<
         }
 
         // If any fields are not blurred
-        if (!field.instance.state.meta.isBlurred) {
+        if (!field.instance.state.meta.isBlurred && cause === 'submit') {
```

Comment:
Per previous previous convos via Discord, let's move this logic out of this block and into the `onChange` function itself so that we don't have to specially edgecase the `cause === 'submit'` behavior here:

https://discord.com/channels/719702312431386674/1263506167901065246/1281270529289883801

Reply:
@crutchcorn I'm not sure where to move this… 

I can't find an `onChange` function. Do I need to create it somewhere?

Reply:
I think I figured out what you meant, could you please re-check - thanks!